spring-cloud-app-broker-docs/src/docs/asciidoc/service-bindings.adoc

```diff
@@ -1,6 +1,14 @@
 [[service-bindings]]
 == Service Bindings
 
-=== Service Binding Creation
+By default, Spring Cloud App Broker does not include functionality for managing bindings to its service instances. App Broker provides interfaces which service broker authors can implement to control service bindings.
 
-=== Service Binding Deletion
+//TODO: Move App Broker API docs base to AsciiDoc attribute.
+
+=== Creating a Service Binding
+
+The service broker application must implement the https://docs.spring.io/spring-cloud-app-broker/docs/1.0.0.BUILD-SNAPSHOT/api/org/springframework/cloud/appbroker/service/CreateServiceInstanceAppBindingWorkflow.html[`CreateServiceInstanceAppBindingWorkflow`] interface.
```

Comment:
The service broker *can* implement this interface to support bindings, but it doesn't have to. It could choose to implement the interfaces from SC OSB instead (https://docs.spring.io/spring-cloud-open-service-broker/docs/current/reference/html5/#service-bindings). This App Broker interface is recommended for consistency with the service instance create/update/delete workflow design.